ReactAndroid/src/main/third-party/android/support/v4/BUCK

```diff
@@ -1,54 +1,131 @@
 load("//tools/build_defs:fb_native_wrapper.bzl", "fb_native")
+load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep")
```

Comment:
Can you add "rn_prebuilt_jar" here?

load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep", "rn_prebuilt_jar")